src/lib/Sendbird.tsx

```diff
@@ -242,7 +243,10 @@ const Sendbird = ({
             compressionRate: 0.7,
             ...imageCompression,
           },
-          isReactionEnabled,
+          isReactionEnabled: getIsReactionEnabled({
+            appLevel: sdkStore?.sdk?.appInfo?.useReaction,
+            globalLevel: isReactionEnabled,
+          }),
```

Comment:
@AhyoungRyu This is related to the feature configuration, I think

Reply:
What's the difference between appLevel <-> globalLevel here?

Reply:
appLevel means the value which is set remotely (ex. from dash board)
and globalLevel means it's set through SendbirdProvider

Reply:
Do we have more configs under appInfo like `useReaction`? If so, should we handle other configs in appInfo too in this way?

Reply:
No, as Airen said we don't need to consider the appInfo anymore.
I will remove the line